Makefile

```diff
@@ -380,7 +380,11 @@ istioctl-install:
 .PHONY: test localTestEnv test-bins
 
 # Run coverage tests
-test: pilot-test mixer-test security-test broker-test galley-test common-test
+JUNIT_UNIT_TEST_XML ?= junit_unit_tests.xml
+test:
+	set -o pipefail; \
```

Comment:
mkdir -p $(notdir $(JUNIT_UNIT_TEST_XML))

Reply:
I think you meant to use dir instead of notdir ?

Reply:
oh, yeah, sorry.